src/samples/Uno.Material.Samples/Uno.Material.Samples.Shared/MarkupExtensions/VisibleIndex.cs

```diff
@@ -0,0 +1,26 @@
+﻿using System;
+using Windows.UI.Xaml;
+using Windows.UI.Xaml.Data;
+using Windows.UI.Xaml.Markup;
+
+namespace Uno.Material.Samples.MarkupExtensions
+{
```

Comment:
Is this too fancy for a sample? :D

Reply:
it's cleaner than having to declare an actual converter